packages/edit-site/src/components/header/index.js

```diff
@@ -89,7 +91,10 @@ export default function Header( {
 		<div className="edit-site-header">
 			<div className="edit-site-header_start">
 				<MainDashboardButton.Slot>
-					<FullscreenModeClose />
+					<DashboardButton
+						isOpen={ isNavigationOpen }
+						onClick={ onToggleNavigation }
+					/>
```

Comment:
I've got a few doubts about this change.
While it's true that this button doesn't close fullscreen mode anymore, it also doesn't navigate back to the dashboard (so the "Open dashboard" label in the component is misleading) — and it's not even technically _just_ a button 😄.

What it does now is to toggle the navigation sidebar, so I'd be more inclined in calling it `NavigationPanelToggle`.

---

I think we could also keep `FullscreenModeClose`.
While it would be left unused in this PR, we could update it to become (_be_?) the actual "dashboard button".
Consumers relying on it would not break, and we could use it inside the navigation toggle, or in the navigation itself (or wherever we see fit).

Either way, my opinion is that we should probably avoid removing existing components from an experimental PR whose purpose is to be a starting point, and not the finished product.

Reply:
I agree that that name is more appropriate. Also no objections to keeping the `FullscreenModeClose` if we are going to repurpose it and include it in navigation.

Reply:
My point is also that the navigation sidebar is still in flux, and I'm not super keen on removing an established component (albeit experimental).
We might end up not using it, or heavily modify it, but let's care about that when we'll be actually working on the back navigation. 🙂

Reply:
- Renamed `DashboardButton` to `NavigationToggle`
- Reverted `FullscreenModeClose` component, let's keep it for now and we will iterate upon it